grafana-ocp/templates/grafana.yaml

```diff
@@ -66,7 +62,7 @@ spec:
       org_role: Admin
     server:
       http_port: "3001"
-      http_addr: localhost      
+      http_addr: 0.0.0.0
```

Comment:
this is wrong, it has to be localhost for the oauth proxy to matter.

Reply:
This seems to be required for the readiness probe to work. I couldn't find a way to change the readiness probe settings created by the operator. Making all port 3001 accessible to grafana fixes this issue.